conda-recipes/llvm14-svml.patch

```diff
@@ -1,8 +1,4 @@
-From bc2dcd190b7148d04772fa7fcd18b5200b758d4a Mon Sep 17 00:00:00 2001
-From: Ivan Butygin <ivan.butygin@gmail.com>
-Date: Sun, 24 Jul 2022 20:31:29 +0200
-Subject: [PATCH] Fixes vectorizer and extends SVML support
-
+Fixes vectorizer and extends SVML support
```

Comment:
Can you retain the attribution by using https://github.com/Hardcode84/llvm-project/commit/9de32f5474f1f78990b399214bdbb6c21f8f098e.patch intead?

Reply:
Added.